src/main/java/pl/agawesolowska/ticketbookingapp/facade/BookingFacade.java

```diff
@@ -0,0 +1,81 @@
+package pl.agawesolowska.ticketbookingapp.facade;
+
+import java.math.BigDecimal;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import pl.agawesolowska.ticketbookingapp.model.dto.BookingRequestDTO;
+import pl.agawesolowska.ticketbookingapp.model.dto.BookingResultDTO;
+import pl.agawesolowska.ticketbookingapp.model.entity.Booking;
+import pl.agawesolowska.ticketbookingapp.model.entity.Customer;
+import pl.agawesolowska.ticketbookingapp.model.entity.Screening;
+import pl.agawesolowska.ticketbookingapp.model.entity.Seat;
+import pl.agawesolowska.ticketbookingapp.service.BookingService;
+import pl.agawesolowska.ticketbookingapp.service.CustomerService;
+import pl.agawesolowska.ticketbookingapp.service.ScreeningService;
+import pl.agawesolowska.ticketbookingapp.service.SeatService;
+import pl.agawesolowska.ticketbookingapp.util.DateTimeUtils;
+import pl.agawesolowska.ticketbookingapp.util.SeatReservationUtils;
+
+/**
+ * @author Aga Wesołowska
+ *
+ */
+@Component
+public class BookingFacade {
```

Comment:
to jest fasada czy serwis?

Reply:
Fasada - osobna klasa, która wywołuje metody z innych klas, w tym przypadku z serwisów.